api/label.go

```diff
@@ -27,5 +27,5 @@ type LabelKeyFind struct {
 // LabelService is the service for labels.
 type LabelService interface {
 	// FindLabelKeys finds all available keys for labels.
-	FindLabelKeys(ctx context.Context, find *LabelKeyFind) ([]LabelKey, error)
+	FindLabelKeys(ctx context.Context, find *LabelKeyFind) ([]*LabelKey, error)
```

Comment:
Maybe `FindLabelKeyList`? Most APIs use `FindSomethingList` rather than `FindSomethings` now, and we should be consistent.